app/build.gradle

```diff
@@ -50,6 +50,7 @@ android {
         multiDexEnabled true
 
         testInstrumentationRunner "com.twilio.video.app.InstrumentationRunner"
+        testInstrumentationRunnerArguments clearPackageData: 'true'
```

Comment:
Does this help with running tests locally?

Reply:
Yeah it helps for local tests and tests on Firebase Test Lab. It basically clears all of the app cache and persisted data in between each test case. You no longer run into issues with login as each test starts in a logged out state.